libcontainerd/container_windows.go

```diff
@@ -289,7 +289,7 @@ func (ctr *container) waitExit(process *process, isFirstProcessToStart bool) err
 					logrus.Error(err)
 				}
 			}
-		}()
+		}(si)
 	}
 
 	logrus.Debugf("libcontainerd: waitExit() completed OK, %+v", si)
```

Comment:
Another fix would be to move this debug statement to before the restart check, I think that would be more clear rather than duplicating `si` in the call to the go func.